### PR TITLE
refactor: codebase check

### PR DESCRIPTION
**Pull request by d-roak**
- rename topology protocol/CRO to DRP
- rm unused stuff
- apply linter suggestions for protobuf
- add biome to package.json
- make drp message protocol string for libp2p global (#244)
- added config objects in the packages that it was missing
- use the same log pattern everywhere
- build dynamically all examples in the CI

### CODE REVIEW

File: packages/node/src/index.ts [hunk 1]
```diff
@@ -48,33 +52,33 @@ export class TopologyNode {
...
+			type: NetworkPb.MessageType.MESSAGE_TYPE_CUSTOM,
 			data,
 		});
-		this.networkNode.sendMessage(peerId, [protocol], message);
+		this.networkNode.sendMessage(peerId, message);
 	}
 
 	async createObject(
-		cro: CRO,
+		cro: DRP,
```
Comment (JanLewDev) [inline packages/node/src/index.ts:75]:
```suggestion
		drp: DRP,
```

File: packages/node/src/index.ts [hunk 2]
```diff
@@ -48,33 +52,33 @@ export class TopologyNode {
...
 		id?: string,
 		abi?: string,
 		sync?: boolean,
 		peerId?: string,
 	) {
-		const object = new TopologyObject(this.networkNode.peerId, cro, id, abi);
+		const object = new DRPObject(this.networkNode.peerId, cro, id, abi);
```
Comment (JanLewDev) [inline packages/node/src/index.ts:81]:
```suggestion
		const object = new DRPObject(this.networkNode.peerId, drp, id, abi);
```

File: packages/node/src/operations.ts [hunk 3]
```diff
@@ -1,10 +1,7 @@
...
-	topologyObjectChangesHandler,
-} from "./handlers.js";
-import type { TopologyNode } from "./index.js";
+import { NetworkPb } from "@ts-drp/network";
+import { type DRPObject, ObjectPb } from "@ts-drp/object";
+import { drpMessagesHandler, drpObjectChangesHandler } from "./handlers.js";
+import { type DRPNode, log } from "./index.js";
 
 /* Object operations */
 enum OPERATIONS {
```
Comment (JanLewDev) [inline packages/node/src/operations.ts:7]: Below is unchanged code still with cro

File: README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <div align="center">
   <img src="https://avatars.githubusercontent.com/u/157637200" height="128">
   <br />
-  <h1>The TypeScript implementation of the Topology Protocol</h1>
+  <h1>The TypeScript implementation of Distributed Replicated Programs (DRP)</h1>
```
Comment (JanLewDev) [inline README.md:4]:
```suggestion
  <h1>The TypeScript implementation of Distributed Replicated Programs (DRPs)</h1>
```

**Review by JanLewDev — COMMENTED** [review]
A few places with old naming

**Review by trungnotchung — APPROVED** [review]

**Pull request merged**